common/scripts/lib/_parseArgs.sh

```diff
@@ -95,28 +132,130 @@ __validate_runtime() {
   if [ "$DB_PASSWORD" == "" ]; then
     __process_msg "DB_PASSWORD is not set"
     __set_db_password
+    setDBPassword=true
   else
     __process_msg "DB_PASSWORD already set, skipping"
   fi
 
-  ################## check for services #################################
-  local services_path=$SCRIPTS_DIR/configs/services.json
-  if [ ! -f "$services_path" ]; then
-    __process_error "Services file $services_path does not exist, exiting"
-    exit 1
-  else
-    __process_msg "Services: $services_path"
-  fi
-
   ################## check system image registry ###############################
   if [ "$PUBLIC_IMAGE_REGISTRY" == "" ]; then
     __process_msg "PUBLIC_IMAGE_REGISTRY is not set"
     __set_public_image_registry
+    setPublicImageRegistry=true
   else
     __process_msg "PUBLIC_IMAGE_REGISTRY already set, skipping"
   fi
 
-  source $ADMIRAL_ENV
+  ################## confirm values #######################################
+  if $setAccessKey || $setSecretKey || $setAdmiralIP || $setDBIP || $setDBPort || $setDBPassword || $setPublicImageRegistry ; then
+    __process_success "Please confirm that the following values are correct:"
```

Comment:
These values are easy to set now, but hard to change later! Please confirm that they are correct: